browser/brave_wallet/brave_wallet_service_delegate_impl.cc

```diff
@@ -63,6 +63,11 @@ void BraveWalletServiceDelegateImpl::IsExternalWalletInitialized(
     IsExternalWalletInitializedCallback callback) {
   std::unique_ptr<ExternalWalletsImporter> importer =
       std::make_unique<ExternalWalletsImporter>(type, context_);
+  // Do not try to init the importer when external wallet is not installed
+  if (!importer->IsExternalWalletInstalled()) {
+    std::move(callback).Run(false);
+    return;
+  }
```

Comment:
FYI this was causing a callback not called error on the network test. 
So the commit is included here.
It was written from @darkdh who is the reviewer.
I am giving a ++ on this commit though.